packages/engine/Source/Scene/Billboard.js

```diff
@@ -22,6 +22,35 @@ import SceneMode from "./SceneMode.js";
 import SceneTransforms from "./SceneTransforms.js";
 import VerticalOrigin from "./VerticalOrigin.js";
 
+/**
+ * @typedef {object} Billboard.ConstructorOptions
+ *
+ * Initialization options for the first param of Billboard constructor
+ *
+ * @property {Cartesian3} position The position of the billboard.
+ * @property {string} [id] A user-defined object to return when the billboard is picked with {@link Scene#pick}.
+ * @property {boolean} [show=true] A boolean Property specifying the visibility of the billboard.
```

Comment:
```suggestion
 * @property {boolean} [show=true] Determines if this billboard will be shown. 
```

Reply:
For this and for all the following properties, please remove the mention of "Property".